vision/google/cloud/vision/face.py

```diff
@@ -42,6 +65,23 @@ def from_api_repr(cls, response):
 
         return cls(roll, pan, tilt)
 
+    @classmethod
+    def from_pb(cls, response):
```

Comment:
Similar comment here as in #2918, the fact that the argument is a `response` is less important than the fact this it's an `Angle`.